google/cloud/bigtable/internal/bulk_mutator_test.cc

```diff
@@ -133,7 +137,8 @@ TEST(BulkMutatorTest, RetryPartialFailure) {
   // isolation, so call MakeOneRequest() twice, for the r1, and the r2 cases.
   for (int i = 0; i != 2; ++i) {
     EXPECT_TRUE(mutator.HasPendingMutations());
-    auto status = mutator.MakeOneRequest(*mock);
+    auto limiter = std::make_shared<bigtable_internal::NoopMutateRowsLimiter>();
```

Comment:
Shared heap allocation of the limiter seems like overkill in these `MakeOneRequest()` calls where a stack allocation would be fine.

Reply:
Yep, you're right. Fixed.